app/models/author.rb

```diff
@@ -1,3 +1,8 @@
 class Author < ActiveRecord::Base
   validates :name, presence: true
+  has_many :books
+
+  def popular_books
```

Comment:
It feels to me that the number displayed should be a responsibility of the presentation level. Thoughts on passing in the limit as a value from the view? (If the design needed changed to show more or less books, then the Rails model itself shouldn't have to change).